packages/orbit-components/src/Popover/components/ContentWrapper.js

```diff
@@ -131,86 +145,112 @@ StyledPopoverPadding.defaultProps = {
 const StyledPopoverContent = styled.div``;
 
 const StyledOverlay = styled.div`
-  display: block;
-  position: fixed;
-  opacity: ${({ shown }) => (shown ? "1" : "0")};
-  top: 0;
-  left: 0;
-  right: 0;
-  width: 100%;
-  height: 100%;
-  background-color: ${({ theme }) => convertHexToRgba(theme.orbit.paletteInkNormal, 60)};
-  transition: ${transition(["opacity"], "normal", "ease-in-out")};
-  z-index: 999;
+  ${({ theme, shown }) => css`
+    display: block;
+    position: fixed;
+    opacity: ${shown ? "1" : "0"};
+    top: 0;
+    left: 0;
+    right: 0;
+    width: 100%;
+    height: 100%;
+    background-color: ${convertHexToRgba(theme.orbit.paletteInkNormal, 60)};
+    transition: ${transition(["opacity"], "normal", "ease-in-out")};
+    z-index: 999;
 
-  ${media.largeMobile(css`
-    display: none;
-  `)};
+    ${mq.largeMobile(css`
+      display: none;
+    `)};
+  `}
 `;
 // $FlowFixMe: https://github.com/flow-typed/flow-typed/issues/3653#issuecomment-568539198
 StyledOverlay.defaultProps = {
   theme: defaultTheme,
 };
 
 const StyledPopoverClose = styled.div`
-  padding: ${({ theme }) => popoverPadding(theme)};
-
-  ${media.largeMobile(css`
+  padding: ${popoverPadding};
+  ${mq.largeMobile(css`
     display: none;
     visibility: hidden;
     padding-bottom: 0;
   `)}
 `;
+
 // $FlowFixMe: https://github.com/flow-typed/flow-typed/issues/3653#issuecomment-568539198
 StyledPopoverClose.defaultProps = {
   theme: defaultTheme,
 };
 
 const PopoverContentWrapper = ({
   children,
-  offset,
   onClose,
   width,
+  offset = { top: 4, left: 0 },
+  referenceElement,
   dataTest,
-  preferredPosition,
-  preferredAlign,
-  containerRef,
+  placement = PLACEMENTS.BOTTOM_START,
   noPadding,
   overlapped,
   shown,
   fixed,
   lockScrolling = true,
   actions,
 }: Props): React.Node => {
+  const [actionsDimensions, setActionsDimensions] = React.useState(0);
   const { isInsideModal } = React.useContext(ModalContext);
   const { isLargeMobile, isTablet } = useMediaQuery();
-  const popover: {| current: React.ElementRef<*> |} = React.useRef(null);
+
   const content: {| current: React.ElementRef<*> |} = React.useRef(null);
   const scrollingElementRef = React.useRef<HTMLElement | null>(null);
   useLockScrolling(scrollingElementRef, lockScrolling && !isLargeMobile);
+
+  const popoverRef: {| current: React.ElementRef<*> |} = React.useRef(null);
```

Comment:
While you're changing refs you can also update how we type them:
```suggestion
  const popoverRef = React.useRef<HTMLElement | null>(null);
```